elements/pfe-card/pfe-card.scss

```diff
@@ -170,7 +170,7 @@ $size-small: (
     // Declare the header background color
     background-color: pfe-local(BackgroundColor, $region: header);
     color:            pfe-local($cssvar: Color, $region: header);
-    :host([on="dark"]) & {
+    :host([color-palette^="dark"]) & {
```

Comment:
seems legit. I wanted to note that this makes sense b/c @coreyvickery specified that cards should reset their context, i.e. it shouldn't matter what the card is 'on', only it's specified palette.